migration.go

```diff
@@ -93,13 +94,13 @@ set -e -v`)
 }
 
 // LoadMigrationsBetweenAnd returns a list of pending Migration <firstFilename..lastFilename]
-func LoadMigrationsBetweenAnd(workdir, firstFilename, lastFilename string) (list []Migration, err error) {
+func LoadMigrationsBetweenAnd(migrationsPath, firstFilename, lastFilename string) (list []Migration, err error) {
 	// collect all filenames
 	filenames := []string{}
 	// firstFilename and lastFilename are relative to workdir.
 	here, _ := os.Getwd()
 	// change and restore finally
-	os.Chdir(workdir)
+	os.Chdir(migrationsPath)
```

Comment:
chdir returns a PathError, shouldn't this be handled? Not sure what will happen if an invalid path is passed. Maybe a panic?